lib/mariaex/protocol.ex

```diff
@@ -443,6 +457,7 @@ defmodule Mariaex.Protocol do
     send_text_query(state, statement) |> text_query_recv(opts, query)
   end
   def handle_execute(%Query{type: :binary} = query, params, opts, state) do
+    query = sanitize_query(query, state)
```

Comment:
We shouldn't need to sanitize on execute, a name is set on prepare so we should look at doing it only in prepare_insert.

Reply:
I need to acquire the `%Mariaex.Protocol{opts: opts}` keyword list to know whether to use named or unnamed prepared statements. I think that I can't access it in `prepare_insert`, do you have a suggestion for this?

Reply:
This relates to: https://github.com/xerions/mariaex/pull/243#discussion_r228374572

Reply:
Resolved.